flow360/examples/tutorial_2D_crm.py

```diff
@@ -1,12 +1,12 @@
 """
-Tutorial_2dcrm meshing example
+tutorial2DCrm meshing example
 """
 
 from .base_test_case import BaseTestCase
 
 
-class Tutorial_2dcrm(BaseTestCase):
-    name = "tutorial_2dcrm"
+class Tutorial2DCRM(BaseTestCase):
+    name = "tutorial2DCrm"
 
     class url:
         geometry = "local://geometry.csm"
```

Comment:
For the geometry file, can you use the link from the original tutorial to download the input file instead of putting it in the python client repo?

Reply:
I think Maciej should know how to upload those files to aws.